listenbrainz/spark/spark_reader.py

```diff
@@ -103,7 +103,7 @@ def start(self):
         """
 
         with self.app.app_context():
-
+            current_app.logger.info('Spark consumer has started!')
```

Comment:
I like this standard "container has started" message. Should we have a "container exited because FOO" message as well?

Reply:
We have this message, printed by the runit `finish` script:

`web_1               | 2021-03-09 18:00:29 UTC api compat terminated. Exit status 0 (consul triggered reload or service was stopped)`

It's difficult to get a hook in the right place in python to print such a message.